presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1558,13 +1643,23 @@ schema evolution, such as adding, dropping, and renaming columns. With schema
 evolution, users can evolve a table schema with SQL after enabling the Presto
 Iceberg connector.
 
+Presto C++ Support
+^^^^^^^^^^^^^^^^^^
+
+Schema Evolution is supported in Presto C++.
+
 Parquet Writer Version
 ----------------------
 
 Presto now supports Parquet writer versions V1 and V2 for the Iceberg catalog.
 It can be toggled using the session property ``parquet_writer_version`` and the config property ``hive.parquet.writer.version``.
 Valid values for these properties are ``PARQUET_1_0`` and ``PARQUET_2_0``. Default is ``PARQUET_1_0``.
 
+Presto C++ Support
+^^^^^^^^^^^^^^^^^^
+
+Presto C++ supports Parquet writer versions V1.
```

Comment:
This is a little confusing because earlier the properties section stated that writes are not supported?

Reply:
Thanks for pointing it point.
It's about actually reading, so I think I can rephrase it something like - `Presto C++ supports reading Parquet data written with Parquet writer version V1.` ?